lighthouse-core/scripts/compare-timings.js

```diff
@@ -188,22 +238,24 @@ function filter(results) {
 }
 
 /**
- * @param {number=} value
+ * @param {number|string|undefined} value
```

Comment:
why was string added here? it makes the type guard not make sense. maybe change body to `typeof value === 'number'`